src/praisonai/praisonai/cli/commands/ui.py

```
@@ -37,6 +37,71 @@ def _ensure_default_app() -> Path:
+    app_file: Optional[str],
+    reload: bool,
+    ui_name: str
+) -> None:
+    """Common function to launch aiui apps."""
+    # 1. Check praisonaiui is installed
+    try:
+        import importlib.util
+        if importlib.util.find_spec("praisonaiui") is None:
+            raise ImportError
+    except ImportError:
+        print(f"\n\033[91mERROR: PraisonAI UI (aiui) is not installed.\033[0m")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)